src/haxeLanguageServer/server/HaxeServer.hx

```diff
@@ -205,7 +205,6 @@ class HaxeServer {
 		context.callHaxeMethod(ServerMethods.Configure, {
 			noModuleChecks: true,
 			print: context.config.displayServer.print,
-			populateCacheFromDisplay: context.config.user.populateCacheFromDisplay,
```

Comment:
I don't think a lot of people have set `populateCacheFromDisplay` to anything other than default, and it's only been in for a year, but does it have any effects when running against versions of Haxe that supported that flag (or was it only ever in nightlies)?

Reply:
It did do something on `4.3.x`, though it has never been documented (in both vshaxe and Haxe compiler) because it was just a temporary workaround to avoid breaking totally in some projects, before we fix the actual issues (which are now fixed and `populateCacheFromDisplay` has been removed from compiler).

I suppose the actual fix there would be to add Json RPC diagnostics to Haxe `4.3.5`... I can check if it's feasible.

Reply:
if 4.3.5 can get these fixes then I think it would be fine to drop that option. otherwise 4.3.x people would be stuck with LSP issues with no way to enable a workaround.

Reply:
well, if we can't get those fixes into a potential 4.3.5 release, then we might have to add something like
```
static inline var ConfigureWithPopulate = new HaxeRequestMethod<ConfigureParams & {var ?populateCacheFromDisplay:Bool}, Response<NoData>>("server/configure");
```
to `HaxeServer.hx` (or somewhere else)

or live with those projects having to turn off diagnostics, if they are stuck on Haxe 4.3.x.

or recommend they stay on current version of vshaxe. I believe you can choose to install an older version. probably the easiest solution for that problem.

Reply:
I'll see if Json RPC diagnostics can be added to 4.3.5 because that'd be a good thing to have there regardless, but other than that I'm really not sure we should go out of our way to support this weird hack (which fixes a niche problem that usually will hit other issues with compilation server anyway on 4.3) that is likely not even used anywhere with 4.3.x

Reply:
ok, I'll trust your judgement.